### ISSUE TITLE
Integrate package versioning playground tool into NuGet.org

### ISSUE THREAD

**Issue by chgill-MSFT** [NuGet/NuGetGallery#7917]
**Problem:** Figuring out which versions of a package a version range or floating version will resolve to can be confusing at times. Currently, there is not a publicly available tool to help customers figure this out.

**Solution:** @joelverhagen has a neat web app that does exactly this, but may need some work regarding scalability and UI before being featured on NuGet.org. 

[Web App](https://nugettoolsdev.azurewebsites.net/5.5.0-floating.7611/find-best-version-match?versionRange=1.*-*&versions=1.0.0%0D%0A2.0.0-test%0D%0A%0D%0A)

**Comment by nkolev92** [comment]
Duplicate of https://github.com/NuGet/NuGetGallery/issues/7828. 
That's the link that the customers were given...so maybe add context there and close this.

**Comment by JonDouglas** [comment]
TL;DR: https://semver.npmjs.com/ for NuGet

**Comment by nkolev92** [comment]
#7828 basically calls it that :D

**Comment by nkolev92** [comment]
Might be nice to close the newer one @chgill-MSFT

**Comment by joelverhagen** [comment]
This is a duplicate of https://github.com/NuGet/NuGetGallery/issues/7828.
